tests/conftest.py

```diff
@@ -18,6 +16,15 @@ def pytest_generate_tests(metafunc):
         metafunc.parametrize('webdriver', webdrivers)
 
 
+def pytest_exception_interact(node, call, report):
+    # import pdb; pdb.set_trace()
+    if report.failed:
+        # Retrieve test's browser logs through the webdriver
+        print('')
```

Comment:
I'm thinking about something like this instead:

```python
# Retrieve test's browser logs through the webdriver
s = '\n'.join([line for line in node.funcargs['webdriver'].get_log('browser')])
if len(s) > 0:
    print('=== LOGS ===')
    print(s)
```

Reply:
Included :)